public/test/core/redux/reduxTester.ts

```diff
@@ -38,36 +29,57 @@ export interface ReduxTesterThen<State> {
 }
 
 export interface ReduxTesterArguments<State> {
-  preloadedState?: PreloadedState<CombinedState<NoInfer<State>>>;
+  preloadedState?: Partial<NoInfer<State>>;
   debug?: boolean;
 }
 
 export const reduxTester = <State>(args?: ReduxTesterArguments<State>): ReduxTesterGiven<State> => {
   const dispatchedActions: AnyAction[] = [];
-  const logActionsMiddleWare: Middleware<{}, Partial<StoreState>> =
-    (store: MiddlewareAPI<Dispatch, Partial<StoreState>>) => (next: Dispatch) => (action: AnyAction) => {
-      // filter out thunk actions
-      if (action && typeof action !== 'function') {
-        dispatchedActions.push(action);
-      }
+  const logActionsMiddleWare: Middleware<{}, Partial<StoreState>> = (store) => (next) => (action) => {
+    // filter out thunk actions
+    if (action && typeof action !== 'function') {
+      dispatchedActions.push(action as AnyAction);
```

Comment:
by default, actions are typed as unknown now. that's great for anything in production, but it would mean every test would have to typeguard on expectations now 😬 

this just casts it back to `AnyAction`. open to better suggestions 😄